src/java/main/org/apache/zookeeper/server/SocketUtil.java

```diff
@@ -0,0 +1,63 @@
+package org.apache.zookeeper.server;
```

Comment:
Please add Apache License Header - the Jenkins build will also fail if missing.

Reply:
Will do.